doc/connectivity/resting_state_networks.rst

```diff
@@ -53,20 +53,26 @@ object.
 Visualizing the results
 ========================
 
-We can visualize the components as in the previous examples.
+We can visualize the components as in the previous examples. The first plot
+shows a map generated from all the components. Then we plot an axial cut for
+each component separately.
 
 .. literalinclude:: ../../examples/connectivity/plot_canica_resting_state.py
     :start-after: ### Visualize the results #####################################################
 
-.. |left_img| image:: ../auto_examples/connectivity/images/plot_canica_resting_state_002.png
+.. image :: ../auto_examples/connectivity/images/plot_canica_resting_state_001.png
+   :align: center
    :target: ../auto_examples/connectivity/plot_canica_resting_state.html
-   :width: 48%
 
-.. |right_img| image:: ../auto_examples/connectivity/images/plot_canica_resting_state_003.png
+.. |left_img| image:: ../auto_examples/connectivity/images/plot_canica_resting_state_003.png
    :target: ../auto_examples/connectivity/plot_canica_resting_state.html
-   :width: 48%
+   :width: 33%
```

Comment:
I asked for that. The single component figures looked bigger than the summary figure, and that looked wrong. 

Sent from my phone. Please forgive brevity and mis spelling



On Jul 17, 2015, 16:58, at 16:58, "Loïc Estève" <notifications@github.com> wrote:
>>     :target:
>../auto_examples/connectivity/plot_canica_resting_state.html
>> -   :width: 48%
>> +   :width: 33%
>
>Interesting you had to tweak the width of the figures!
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nilearn/nilearn/pull/704/files#r34897053